huaweicloud/data_source_huaweicloud_obs_bucket_object.go

```diff
@@ -0,0 +1,144 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/huaweicloud/golangsdk/openstack/obs"
+)
+
+func dataSourceObsBucketObject() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceObsBucketObjectRead,
+
+		Schema: map[string]*schema.Schema{
+			"body": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
the recommended order in Schema: Required --> Optional --> Computed

Reply:
fixed